packages/form-js-viewer/package.json

```diff
@@ -46,6 +46,7 @@
     "big.js": "^6.2.1",
     "classnames": "^2.3.1",
     "didi": "^9.0.0",
+    "feelers": "^0.1.0-alpha.8",
```

Comment:
If we don't use a peer dependency here, we should add it to the roll-up externals, so it won't be bundled in the distros.

Reply:
Just added it.